.github/workflows/ci.yml

```
@@ -24,13 +24,14 @@ jobs:
 
       - name: 'Run unit tests and other checks'
         run: |
-          source environment
-          _link dev
-          _refresh
-
+          source environment  # load global defaults
           make virtualenv
           source .venv/bin/activate
           make requirements
+          deployment=$(PYTHONPATH=src python scripts/check_branch.py --print)
```

Wouldn't this still work without setting `PYTHONPATH` here, since it's already been set by the call to `source environment`?

```suggestion
          deployment=$(python scripts/check_branch.py --print)
```

---

Yup, it does. Good catch.